client/packages/system/src/Log/api/api.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  SortBy,
+  LogSortInput,
+  LogSortFieldInput,
+} from '@openmsupply-client/common';
+import { Sdk, LogRowFragment } from './operations.generated';
+
+export type ListParams = { recordId: string; sortBy?: SortBy<LogRowFragment> };
+
+const logParsers = {
+  toSortInput: (sortBy: SortBy<LogRowFragment>): LogSortInput => {
+    return { desc: sortBy.isDesc, key: sortBy.key as LogSortFieldInput };
+  },
+};
+
+export const getLogQueries = (sdk: Sdk) => ({
+  get: {
+    listByRecord: async ({ sortBy, recordId }: ListParams) => {
+      const response = await sdk.logs({
+        offset: 0,
+        first: 1000,
```

Comment:
just hard coding these values to bypass pagination, which isn't required on this one.
have implemented sort - but the fields aren't helpful to sort by (IMHO), so have left them out of the UI